tutorials/text/annotations.py

```diff
@@ -348,19 +388,64 @@
 ax.add_artist(box)
 
 ###############################################################################
-# As in the legend, the bbox_to_anchor argument can be set.  Using the
-# HPacker and VPacker, you can have an arrangement(?) of artist as in the
-# legend (as a matter of fact, this is how the legend is created).
+# Another method of anchoring an artist relative to a parent axes or anchor
+# point is via the *bbox_to_anchor* argument of `.AnchoredOffsetbox`. This
+# artist can then be automatically positioned relative to another artist using
+# `.HPacker` and `.VPacker`. ::
+#
+#   from matplotlib.offsetbox import (AnchoredOffsetbox, DrawingArea, HPacker,
+#                                     TextArea)
+#
+#   box1 = TextArea(" Test: ", textprops=dict(color="k"))
+#   box2 = DrawingArea(60, 20, 0, 0)
+#   # see gallery example for ellipse drawing code
+#   box = HPacker(children=[box1, box2], align="center", pad=0, sep=5)
+#   anchored_box = AnchoredOffsetbox(loc='lower left', child=box, pad=0.,
+#                                    frameon=True, borderpad=0.,
+#                                    bbox_to_anchor=(0., 1.02),
+#                                    bbox_transform=ax.transAxes)
 #
 # .. figure:: ../../gallery/userdemo/images/sphx_glr_anchored_box04_001.png
 #    :target: ../../gallery/userdemo/anchored_box04.html
 #    :align: center
 #
-# Note that unlike the legend, the ``bbox_transform`` is set
-# to IdentityTransform by default.
+# Note that, unlike in `.Legend`, the ``bbox_transform`` is set to
+# IdentityTransform by default. The full example can be found at
+# :doc:`/gallery/userdemo/anchored_box04`.
 #
-# Coordinate systems for Annotations
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Defining custom box styles
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# You can use a custom box style. The value for the ``boxstyle`` can be a
+# callable object in the following forms. ::
+#
+#         def __call__(self, x0, y0, width, height, mutation_size,
+#                      aspect_ratio=1.):
```

Comment:
`__call__` is for the class way, but that's not described here, just the plain callable:
```suggestion
#         def custom_box_style(x0, y0, width, height, mutation_size):
```
(also `aspect_ratio` seems to be gone now?)